core/src/main/kotlin/run/qontract/core/utilities/Utilities.kt

```diff
@@ -253,29 +253,41 @@ fun contractFilePathsFrom(configFilePath: String, workingDirectory: String, sele
     val sources = loadSources(configFilePath)
 
     return sources.flatMap { source ->
-        val repoDir = when(source) {
-            is GitRepo -> {
-                println("Looking for contracts in local environment")
-                val userHome = File(System.getProperty("user.home"))
-                val defaultQontractWorkingDir = userHome.resolve(".qontract/repos")
-                val defaultRepoDir = source.directoryRelativeTo(defaultQontractWorkingDir)
-
-                when {
-                    defaultRepoDir.exists() && SystemGit(defaultRepoDir.path).workingDirectoryIsGitRepo() -> {
-                        println("Using local contracts")
-                        defaultRepoDir
-                    }
-                    else -> {
-                        val reposBaseDir = File(workingDirectory).resolve("repos")
-                        println("Couldn't find local contracts, cloning ${source.gitRepositoryURL} into ${reposBaseDir.path}")
-                        if(!reposBaseDir.exists())
-                            reposBaseDir.mkdirs()
+        println("Looking for contracts in local environment")
```

Comment:
We may not need to check the local environment if we know the repo is a mono repo. We can pair on this.

Reply:
Done.